htmd/molecule/voxeldescriptors.py

```diff
@@ -149,34 +149,41 @@ def _findDonors(mol, bonds):
 
 
 def _getSigmas(mol):
-    """ Returns the VDW radii of each atom in the molecule
+    # Sigmas taken from VMD vdW radii.
```

Comment:
I don't know if we are going for chemical correctness here, but lennard-jones sigmas and van der waals radius are not the same thing